deepchecks/vision/checks/model_evaluation/single_dataset_performance.py

```diff
@@ -0,0 +1,173 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing a check for computing a scalar performance metric for a single dataset."""
+from numbers import Number
+from typing import Dict, List
+
+from ignite.metrics import Metric
+
+from deepchecks.core import CheckResult, ConditionResult, DatasetKind
+from deepchecks.core.checks import ReduceMixin
+from deepchecks.core.condition import ConditionCategory
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.vision import Batch, Context, SingleDatasetCheck
+from deepchecks.vision.metrics_utils.metrics import get_scorers_dict, metric_results_to_df
+
+__all__ = ['SingleDatasetPerformance']
+
+
+class SingleDatasetPerformance(SingleDatasetCheck, ReduceMixin):
+    """Calculate a performance metric as a scalar for a given model and a given dataset.
+
+    Parameters
+    ----------
+    alternative_scorers : Dict[str, Callable], default: None
```

Comment:
Would name this scorers (as in tabular performance)

Reply:
@Nadav-Barak renamed